frontend/src/lib/CheckStatus.tsx

```diff
@@ -0,0 +1,12 @@
+type Props = {
```

Comment:
will we add more function/component into this file ?
if yes, should this rename to ``CheckStatusProps`` and rename filename too ?

Reply:
if this file is only for `status`, since the stats only has three type,  error, success and pending. i think this is okay for now.
currently, i have no idea what other things we may encounter. but if it's far more than `status` maybe new file would be good in my opinion.

Reply:
Maybe i can move my ``RouteOnNotOK`` function to here? cus it's kind of similar.
But if u think that's bad, ill also give ``RouteOnNotOK`` a file and leave this as it is.

Reply:
i think it's okay. i can change the props name to `CheckStatusProps`, and since the two status is different, i'll change the file name to `Status.ts`
will this be better?